StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Controllers/PayWithLinkWebController.swift

```diff
@@ -17,20 +17,14 @@ import AuthenticationServices
 @available(macCatalystApplicationExtension, unavailable)
 protocol PayWithLinkWebControllerDelegate: AnyObject {
 
-    func payWithLinkWebControllerDidConfirm(
+    func payWithLinkWebControllerDidComplete(
         _ payWithLinkWebController: PayWithLinkWebController,
         intent: Intent,
-        with paymentOption: PaymentOption,
-        completion: @escaping (PaymentSheetResult) -> Void
+        with paymentOption: PaymentOption
```

Comment:
There is no result, as this function just creates a PaymentMethod instead of confirming the PaymentSheet. (Which is why the name is now `didComplete` instead of `didConfirm`).